src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/WellKnownTypeValidationTests.vb

```diff
@@ -742,7 +742,10 @@ End Namespace
                          WellKnownMember.System_Diagnostics_CodeAnalysis_UnscopedRefAttribute__ctor,
                          WellKnownMember.System_Runtime_CompilerServices_MetadataUpdateOriginalTypeAttribute__ctor,
                          WellKnownMember.System_Runtime_CompilerServices_RuntimeHelpers__CreateSpanRuntimeFieldHandle,
-                         WellKnownMember.System_Runtime_CompilerServices_RequiresLocationAttribute__ctor
+                         WellKnownMember.System_Runtime_CompilerServices_RequiresLocationAttribute__ctor,
+                         WellKnownMember.System_Collections_Generic_IReadOnlyList_T__Item,
+                         WellKnownMember.System_Collections_Generic_IList_T__Item,
+                         WellKnownMember.System_Collections_Generic_List_T__Item
```

Comment:
I didn't understand why only these indexers had to be added here.

Reply:
Good catch. It turns out these members are not found because the well-known member name is "this[]" which matches the C# name rather than "Item" expected in VB.